src/main/scala/study/softserve/scala/Application.scala

```diff
@@ -1,39 +1,48 @@
 package study.softserve.scala
 
 import akka.actor.{ActorRef, Props}
+import akka.kafka.ProducerSettings
+import akka.kafka.scaladsl.SendProducer
 import akka.pattern.ask
 import akka.routing.RoundRobinPool
 import akka.util.Timeout
-import study.softserve.scala.JSONParser.CityWeather
+import com.myprotos.myproto.CityWeather
+import com.typesafe.config.ConfigFactory
+import org.apache.kafka.clients.producer.{ProducerRecord, RecordMetadata}
+import org.apache.kafka.common.serialization.{ByteArraySerializer, StringSerializer}
 
-import scala.concurrent.ExecutionContextExecutor
 import scala.concurrent.duration.DurationInt
+import scala.concurrent.{ExecutionContextExecutor, Future}
 import scala.language.{implicitConversions, postfixOps}
 
 object Application {
 
   def main(args: Array[String]): Unit = {
+    implicit val ec: ExecutionContextExecutor = system.dispatcher
+
+    val config = ConfigFactory.load()
+    val producerConfig = config.getConfig("akka.kafka.producer")
+    val producerSettings: ProducerSettings[String, Array[Byte]] =
+      ProducerSettings(producerConfig, new StringSerializer, new ByteArraySerializer)
+    val producer = SendProducer(producerSettings)
```

Comment:
Let's put producer's code into separate module, and make a nice function. Also please make topic names configurable.